scripts/build_and_test.sh

```diff
@@ -15,9 +15,9 @@ BUILD_AND_TEST_TEST_SUITE_ID=771f3f55-7db9-11e5-80ff-22000b5181e9
 start_test_suite ${BUILD_AND_TEST_TEST_SUITE_ID}
  
 function build_and_test() {
-  ${MAVEN} clean deploy -s "${MAVEN_SETTINGS}"
+    ${MAVEN} clean package -s "${MAVEN_SETTINGS}"
```

Comment:
nit: Is this script used by external contributors as well? I wonder if we should start adding a differentiation between internal and external builds where they share the same core but things like `start_test_suite ${BUILD_AND_TEST_TEST_SUITE_ID}` should only be in the internal CI script.